pkg/function/server/server.go

```diff
@@ -62,4 +64,19 @@ func (s *server) Start(inputOptions ...Option) {
 	if err := grpcSvr.Serve(lis); err != nil {
 		log.Fatalf("failed to start the gRPC server: %v", err)
 	}
+
+	httpfuncsdk.Start(context.Background(), func(ctx context.Context, key, msg []byte) (httpfuncsdk.Messages, error) {
```

Comment:
It can not be executed when gRPC server is started, use a goroutine to start it before starting the gRPC server.